src/components/create/PartBox/PartBox.jsx

```diff
@@ -25,17 +33,20 @@ const PartBox = () => {
     { id: 7, value: "notyet", label: "미정" }
   ];
 
-  const [channel, setChannel] = React.useState("front");
-  const [people, setPeople] = React.useState("1");
+  const [channel, setChannel] = React.useState(initialChannel);
+  const [people, setPeople] = React.useState(initialPeople);
   const [stackOptions, setStackOptions] = React.useState(
     SKILL_OPTIONS[0].options
   );
   const [stacks, setStacks] = React.useState(null);
 
   React.useEffect(() => {
-    console.log("channel", channel);
-    console.log("people", people);
-    console.log("stacks", stacks);
+    const part = {
+      channel: channel,
+      people: people,
+      skills: stacks
+    };
+    handleUpdate(id, part);
```

Comment:
크게 중요한 부분은 아닌데 `stacks`를 `skills`로 맞출 수 있다면 축약할 수 있을 것 같아요.

```suggestion
    handleUpdate(id, {
      channel,
      people,
      skills
    });
```

Reply:
와.. 감사합니당 그렇네용 !!